### DISCUSSION TITLE
No vehicles in spawn vehicle

### DISCUSSION THREAD

**Discussion by USBMenus** [YimMenu/YimMenu#1654] [Q&A]
idk if im missing a file or folder or something but it says no vehicles in registry
![image](https://github.com/YimMenu/YimMenu/assets/69381115/b994a5d7-eb79-4ad0-bd43-e59bcdc6285d)

**Comment by xiaoxiao921** [comment]
Can you give the %appdata%/YimMenu/cout.log file?

**Reply by USBMenus** [reply]
i am looking for something in it, it says loaded 0 vehicles from cache
